__init__.py

```diff
@@ -413,12 +422,12 @@ def handle_automation_intent(self, message):
         elif "script" in ha_entity['id']:
             self.speak_dialog('homeassistant.automation.trigger',
                               data={"dev_name": ha_entity['dev_name']})
-            self.ha.execute_service("homeassistant", "turn_on",
+            self.ha.execute_service("script", "turn_on",
```

Comment:
Was this a change in the HA API or a bug in the Skill?

Reply:
Not sure when it was introduced in API but I just wanted to use correct service. You can probably use `homeassistant` for everything, but it's not nice, when there is service specially for it.

Reply:
It's also same for `scene` in line 430.